addon/components/tool-tipster.js

```diff
@@ -84,8 +84,8 @@ export default Ember.Component.extend({
       }
     });
     options.trigger = this.get('triggerEvent');
-    // Handle SafeString
-    if (content instanceof SafeString) {
+    // Handle htmlSafe
+    if (content instanceof htmlSafe) {
       options.content = content.toString();
```

Comment:
Hmm this will not work. If you create a string using `htmlSafe("<div>someValue</div>")` it will still be an instance of `SafeString`. And the new check `String.isHTMLSafe` will work only in Ember v2.8 and up but will not work with 2.7 or lower versions.